port/common/omrsysinfo_helpers.c

```diff
@@ -433,16 +498,27 @@ omrsysinfo_get_x86_description(struct OMRPortLibrary *portLibrary, OMRProcessorD
 	desc->features[1] = CPUInfo[CPUID_ECX];
 	desc->features[2] = 0;
 
-	/* If OSXSAVE is supported, populate the XSAVE state in desc->features[2]. Unused bits (4-31) are reserved for future expansion. */
+	/* If OSXSAVE is supported, populate the XSAVE state in desc->features[2], bits 0-3 */
```

Comment:
missing punctuation. let's fix such cases in all new and modified code.
```suggestion
	/* If OSXSAVE is supported, populate the XSAVE state in desc->features[2], bits 0-3. */
```